key,default_value
userAgent,"aws-cli/1.11.29 Python/2.7.12 Linux/4.4.41-36.55.amzn1.x86_64 botocore/1.4.86"
eventID,"63d55b2e-4f3f-4a01-a534-998eb17283c"
errorMessage,"The instance ID 'i-3cffbc130fddab6f7' does not exist"
userIdentity_type,"IAMUser"
userIdentity_principalId,"AIDA9BO36HFBHKGJAO9C1"
userIdentity_arn,"arn:aws:iam::811596193553:user/backup"
userIdentity_accountId,"811596193553"
userIdentity_accessKeyId,"AKIA1ZBTOEKWKVHP6GHZ"
userIdentity_userName,"backup"
eventType,"AwsApiCall"
errorCode,"Client.InvalidInstanceID.NotFound"
sourceIPAddress,"132.231.4.86"
eventName,"AttachVolume"
eventSource,"ec2.amazonaws.com"
recipientAccountId,"811596193553"
requestParameters_volumeId,"vol-8e100f305b7a6fef3"
requestParameters_instanceId,"i-3cffbc130fddab6f7"
requestParameters_device,"/dev/sdf"
requestParameters_deleteOnTermination,"False"
awsRegion,"us-west-2"
requestID,"ca78f6fc-ca5d-404a-88b5-775423c24d4"
responseElements,"None"
eventVersion,"1.05"
eventTime,"2017-03-01T20:36:09Z"
